recipes/cppyy-cling/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "cppyy-cling" %}
+{% set version = "6.15.2.9" %}
+# The first three items of version define the version of ROOT that we're building from.
+{% set root_version = "6.15.02" %}
+{% set clang_version = "5.0.0" %}
+{% set clang_patches_version = "6.16.00" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://bitbucket.org/wlav/cppyy-backend/get/{{ name }}-{{ version }}.tar.gz
+    sha256: 7b4343d8d68442a80486165d40dbc8b03b0a3299aa891b3f0f83b186e267f2fc
+    patches:
+    - manylinux.patch
+    - skip_download.patch
+  - url: https://root.cern.ch/download/root_v{{ root_version }}.source.tar.gz
+    sha256: 7b4343d8d68442a80486165d40dbc8b03b0a3299aa891b3f0f83b186e267f2fc
+    folder: cling/root-{{ root_version }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - python
+    - pip
+    - zlib
+    - clangdev={{ clang_version }}
```

Having a space here is important for conda-forge to work correctly:
```suggestion
    - clangdev ={{ clang_version }}
```